FsSonarRunner/FsSonarRunnerCore/FsLintRunner.fs

```diff
@@ -19,7 +19,14 @@ type FsLintRule(name : string, value : string) =
 type SonarRules() =
 
     let fsLintProfile =
-        let resourceManager = new ResourceManager("Text" ,Assembly.Load("FSharpLint.Core"))
+        let resourceManager =

```

Comment:
@jmecosta more complex, but I didn't got it runing otherwise. Maybe you know how to simplify it again?